ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -314,7 +314,8 @@ public enum TargetDotNetFrameworkVersion
         Version46 = 7,
         Version461 = 8,
         Version462 = 10,
-        VersionLatest = 10,
+        Version47 = 11,
+        VersionLatest = 11,
```

Comment:
This is a breaking change - and has broken teams (the Roslyn team) the last time this occurred. This pattern should be stopped.